examples/custom-resource/libraries/foobar.rb

```diff
@@ -0,0 +1,14 @@
+class Foobar < Inspec.resource(1)
+  name 'foobar'
+
+  example "
+      describe foobar do
+        its('info') { should eq 'foobar' }
+      end
+  "
+
+  def info
```

Comment:
Here, I'd suggest something like `whoami_output` or something.  Make it clearer that we're doing a neat trick - calling a core resource from a custom one.